src/ontology/config/prefixes.csv

```diff
@@ -224,8 +224,15 @@ PLANP,http://purl.obolibrary.org/obo/PLANP_
 CARO,http://purl.obolibrary.org/obo/CARO_
 NPO,http://purl.bioontology.org/ontology/npo#NPO_
 ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
+ICD10CM2,https://icd.codes/icd10cm/
 ICD10WHO,https://icd.who.int/browse10/2019/en#/
+ICD10WHO2010,http://apps.who.int/classifications/icd10/browse/2010/en#/
```

Comment:
## ICD10WHO
<details><summary>Details</summary>
<p>

I don't remember where, but I saw 2 different valid variations, so I added the missing one:
```csv
ICD10WHO,https://icd.who.int/browse10/2019/en#/
ICD10WHO2010,http://apps.who.int/classifications/icd10/browse/2010/en#/
```

</p>
</details>